docs/api/covidcast-signals/google-symptoms.md

```diff
@@ -13,27 +13,27 @@ grand_parent: COVIDcast Epidata API
 * **Date of last change:** Never
 * **Available for:** county, MSA, HRR, state (see [geography coding docs](../covidcast_geography.md))
 * **Time type:** day (see [date format docs](../covidcast_times.md))
-* **License:** [CC BY](../covidcast_licensing.md#creative-commons-attribution)
+* **License:** To download or use the data, you must agree to the Google [Terms of Service](https://policies.google.com/terms)
```

Comment:
Is this text acceptable? we try to keep the header short, but if the extra language is required, we can make an exception.

```suggestion
* **License:** Google's [Terms of Service](https://policies.google.com/terms)
```

Reply:
Unfortunately it is required. We kept consistent messaging on the license across all of our surfaces.